lib/stitches/version.rb

```diff
@@ -1,3 +1,3 @@
 module Stitches
-  VERSION = '3.7.3'
+  VERSION = '3.8.3'
```

Comment:
Our pull request template contained some outdated instructions for updating the `VERSION`, so doing this is unnecessary. If this PR is approved and merged, one of the maintainers would take care of this when releasing a new version of this gem.

On a side note, the version here should be `3.8.0` following SEMVER guidelines. The `patch` portion of the version (the last digit) gets reset to `0`. If this had been a major version change, it would be `4.0.0`.